util/cron/test-perf.cray-cs-hdr.champs-llvm.bash

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+#
+# Run CHAMPS testing on a cray-cs with HDR IB and LLVM backend
+
+CWD=$(cd $(dirname $0) ; pwd)
+
+export CHPL_TEST_PERF_CONFIG_NAME='16-node-cs-hdr'
+export CHPL_NIGHTLY_TEST_CONFIG_NAME="perf.cray-cs-hdr.champs-llvm"
+export CHPL_TEST_PERF_DESCRIPTION='llvm'
+
+source $CWD/common-champs.bash
+
+source /cy/users/chapelu/setup_system_llvm.bash
+export CHPL_LLVM=system
```

Comment:
Sourcing `common-perf.bash` sources `common.bash`, which should bring in system llvm automatically. You should double check me, but I think you want to reverse this logic where you don't set anything here, but force C backend in the other script maybe by sourcing `common-c-backend.bash` (assuming you want gnu backend and not intel, unsure why intel is needed in `common-champs.bash`)